nextcord/user.py

```diff
@@ -288,8 +304,17 @@ class ClientUser(BaseUser):
         The user's username.
     id: :class:`int`
         The user's unique ID.
+    global_name: :class:`str`
+        The user's display name, if any.
+
+        .. versionadded: 2.6
     discriminator: :class:`str`
-        The user's discriminator. This is given when the username has conflicts.
+        The user's discriminator.
+
+        .. warning::
+            This field is deprecated, and will only return if the user has not yet migrated to the
+            new `username <https://dis.gd/usernames>`_ update.
```

Comment:
Why the underscore here? Is that a RST thing?

Reply:
It is an RST thing, yes